### PR TITLE
remove max_val param from threshold.binary()

### PR DESCRIPTION
**Pull request by afinit**
**Describe your changes**

Removes max_val param from the `threshold.binary` method

**Type of update**

feature enhancement

**Associated issues**

Accomplishes 1 task in this ticket: https://github.com/danforthcenter/plantcv/issues/1194

**Additional context**

There are a few tests failing for me in the `release-4.0`. After these changes, no new tests are failing.

@HaleySchuhl @nfahlgren This morning I sent you both an email about potentially contributing to this project. Figured I'd just push a PR while I waited for a response. Please feel free to mark this up as much as you want to make sure I'm properly contributing.

**For the reviewer**
See [this page](https://plantcv.readthedocs.io/en/latest/pr_review_process/) for instructions on how to review the pull request.
- [ ] PR functionality reviewed in a Jupyter Notebook
- [x] All tests pass
- [x] Test coverage remains 100%
- [x] Documentation tested
- [x] New documentation pages added to `plantcv/mkdocs.yml`
- [X] Changes to function input/output signatures added to `updating.md`
- [ ] Code reviewed
- [ ] PR approved

### CODE REVIEW

File: plantcv/plantcv/threshold/threshold_methods.py [hunk 1]
```diff
@@ -48,7 +46,7 @@ def binary(gray_img, threshold, max_value, object_type="light"):
     params.device += 1
 
     # Threshold the image
-    bin_img = _call_threshold(gray_img, threshold, max_value, threshold_method, "_binary_threshold_")
+    bin_img = _call_threshold(gray_img, threshold, 255, threshold_method, "_binary_threshold_")
```
Comment (afinit) [inline plantcv/plantcv/threshold/threshold_methods.py:49]: happy to update this to keep `max_value` as a variable here and set that to `255` above in the function if that's preferred

**Comment by afinit** [comment]
It looks like there's a build issue. I'm wondering if that's just a "rerun and it should be fine" situation, or somehow related to the fact that I made this pull request from a fork?

**Comment by nfahlgren** [comment]
Hi Mark, you were right, the CI pipeline failed to upload coverage results to CodeCov and that's why it showed as failing. I reran the analysis and it passed this time.

File: plantcv/plantcv/threshold/threshold_methods.py (hunk 1 above)
Comment (HaleySchuhl) [reply plantcv/plantcv/threshold/threshold_methods.py:49]: Hmmm @nfahlgren do we want to remove the argument from the helper function too? I don't imagine we'd ever set it to anything else...

**Review by HaleySchuhl — APPROVED** [review]
Looks good! We might want to remove the `max_value` parameter from the helper function `_call_threshold` in a future PR

**Pull request merged**